docs/team/eesung00.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Siew Ee Sung's Project Portfolio Page
+---
+
+### Project: CinternS
+
+CinternS is a desktop intern-management application used for managing internship applications for Computer Science students. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: to be added soon
+    * What it does: to be added soon
+    * Justification: to be added soon
+    * Highlights: to be added soon
+    * Credits: to be added soon
+
+* **New Feature**: to be added soon
+
+* **Code contributed**: [RepoSense link]()
```

Comment:
```suggestion
* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=eesung00&breakdown=true&sort=groupTitle&sortWithin=title&since=2022-09-16&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
```

Reply:
Thank you for your reminder! 👍